sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/SentenceSentiment.java

```diff
@@ -7,51 +7,61 @@
 import com.azure.core.util.IterableStream;
 
 /**
- * The {@link SentenceSentiment} model that contains a sentiment label of a sentence, confidence score of the
- * sentiment label, length of the sentence and offset of the sentence within a document.
+ * The {@link SentenceSentiment} model that contains a sentiment label of a sentence, confidence scores of the
+ * sentiment label, mined opinions, offset of sentence, and length of sentence within a document.
  */
 @Immutable
 public final class SentenceSentiment {
     private final String text;
-    private final SentimentConfidenceScores confidenceScores;
     private final TextSentiment sentiment;
+    private final SentimentConfidenceScores confidenceScores;
     private final IterableStream<MinedOpinion> minedOpinions;
+    private final int offset;
+    private final int length;
 
     /**
      * Creates a {@link SentenceSentiment} model that describes the sentiment analysis of sentence.
+     *
      * @param text The sentence text.
      * @param sentiment The sentiment label of the sentence.
      * @param confidenceScores The sentiment confidence score (Softmax score) between 0 and 1, for each sentiment label.
-     *   Higher values signify higher confidence.
+     * Higher values signify higher confidence.
      */
     public SentenceSentiment(String text, TextSentiment sentiment, SentimentConfidenceScores confidenceScores) {
         this.text = text;
         this.sentiment = sentiment;
-        this.minedOpinions = null;
         this.confidenceScores = confidenceScores;
+        this.minedOpinions = null;
+        this.offset = 0;
+        this.length = 0;
```

Comment:
These initializations are only on SentenceSentiment and not on `LinkedEntityMatch` or `CategorizedEntity` ?

Reply:
we do have in LinkedEntityMatch: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/LinkedEntityMatch.java#L44

and CategorizedEntity: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/CategorizedEntity.java#L56